rasa/core/domain.py

```diff
@@ -187,13 +207,12 @@ def _get_session_config(session_config: Dict) -> SessionConfig:
     @classmethod
     def from_directory(cls, path: Text) -> "Domain":
         """Loads and merges multiple domain files recursively from a directory tree."""
-        from rasa import data
 
         domain = Domain.empty()
         for root, _, files in os.walk(path, followlinks=True):
             for file in files:
                 full_path = os.path.join(root, file)
-                if data.is_domain_file(full_path):
```

Comment:
Should we delete `data.is_domain_file` now? I only see one other use, in `multi_project.py`. If we update that use then we should be able to delete it (given that this goes into 2.0 it shouldn't be a problem)

Reply:
thanks for spotting that, yes, I'll remove the `data::is_domain_file()`.